tests/migration_test.go

```diff
@@ -2573,6 +2573,96 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				Entry("[sig-compute][test_id:3241]cancel a migration right after posting it", false),
 				Entry("[sig-compute][test_id:3246]cancel a migration with virtctl", true),
 			)
+
+			Context("[Serial]when target pod cannot be scheduled and is suck in Pending phase", func() {
+
+				var nodesSetUnschedulable []string
+
+				BeforeEach(func() {
+					By("Keeping only one schedulable node")
+					schedulableNodes := libnode.GetAllSchedulableNodes(virtClient).Items
+					Expect(len(schedulableNodes)).To(BeNumerically(">=", 1))
+
+					// Iterate on all schedulable nodes but one
+					for _, schedulableNode := range schedulableNodes[:len(schedulableNodes)-1] {
+						tests.SetNodeUnschedulable(schedulableNode.Name, virtClient)
+						nodesSetUnschedulable = append(nodesSetUnschedulable, schedulableNode.Name)
+					}
+				})
+
+				AfterEach(func() {
+					By("Restoring nodes to be schedulable")
+					for _, schedulableNodeName := range nodesSetUnschedulable {
+						tests.SetNodeSchedulable(schedulableNodeName, virtClient)
+					}
+				})
+
+				It("should be able to properly abort migration", func() {
+					By("Starting a VirtualMachineInstance")
+					vmi := tests.NewRandomVMI()
+					vmi = runVMIAndExpectLaunch(vmi, 240)
+					defer func() {
```

Comment:
Please remove this. We wouldn't get artifacts.

Reply:
Done, thanks